test/csuite/schema_abort/main.c

```diff
@@ -148,13 +147,23 @@ static void usage(void) WT_GCC_FUNC_DECL_ATTRIBUTE((noreturn));
 
 /*
  * usage --
- *     TODO: Add a comment describing this function.
+ *     Display usage statement and exit failure.
  */
 static void
 usage(void)
 {
-    fprintf(stderr, "usage: %s [-h dir] [-s stop_timestamp] [-T threads] [-t time] [-BClmvxz]\n",
+    fprintf(stderr, "usage: %s [-h dir] [-s stop_timestamp] [-T threads] [-t time] [-CSlmvxz]\n",
```

Comment:
Please alphabetize S in ClmSvxz.

Reply:
Sure, done in 825328f348d8d139808969ec1ea7907c06c992a3.